test/unit/source/source_cache.test.js

```diff
@@ -724,12 +757,6 @@ test('SourceCache#_updateRetainedTiles', (t)=> {
         stateCache[idealTiles[0].id] = 'loaded';
         const retained = sourceCache._updateRetainedTiles(idealTiles, 1);
         t.deepEqual(getTileSpy.getCalls().map((c)=>{ return c.args[0]; }), [
-            // checks all child tiles to see if they're loaded before loading parent
-            new TileCoord(2, 0, 2),
-            new TileCoord(2, 1, 2),
-            new TileCoord(2, 0, 3),
-            new TileCoord(2, 1, 3),
```

Comment:
At first, I was going to ask whether there's some other way we could preserve testing of this logic, checking for child tiles before trying to load parent...  But, if I'm understanding things right, it looks like the `'prefer loaded child tiles to parent tiles'` at least partially confirms that the right thing is happening, so we're probably okay.  Does that sound right to you, @mollymerp? 

Do you think it would be worthwhile to add a test that's similar to 
the `'prefer loaded child tiles to parent tiles'` one, but where _all_ the child tiles are loaded, to confirm that we don't load the parent in that case?

Reply:
done!